src/scoreboard.js

```diff
@@ -0,0 +1,12 @@
+var models = require('./models');
+
+module.exports = function (req, res, next){
+  models.User.findAll({
+    attributes: ['name', 'score'],
+    order: [ [ 'score', 'DESC' ] ]
+  }).then(scores => {
+    if (scores) res.send(scores);
+    else res.sendStatus(400);
+    next();
```

Comment:
No need to call `next` here when you have already sent the response. The last middleware in our app that get executed after route handling middleware is `error` handling middleware which only get executed when an `next(err)` is there